analyzer/server_easyjson.go

```diff
@@ -1,520 +1,16 @@
-// Code generated by easyjson for marshaling/unmarshaling. DO NOT EDIT.
+// TEMPORARY AUTOGENERATED FILE: easyjson stub code to make the package
```

Comment:
This file was not properly generated by easyjson. You probably only need to restore it.